app/models/users.py

```diff
@@ -0,0 +1,25 @@
+from typing import Union
+from sqlmodel import Field, SQLModel
+
+'''
+Declares the JWT token, token type will reference
+bearer and the access token is generated per user
+encoding via the secret key and HS256
+'''
+class Token(SQLModel):
+    access_token: str
+    token_type: str
+
+'''
+TokenData links the JWT token to the username
+'''
+class TokenData(SQLModel):
+    username: Union[str, None] = None
+
+class Users(SQLModel, table=True):
+    username: str = Field(primary_key=True)
+    hashed_password: Union[str, None] = None
+    email: Union[str, None] = None
+    full_name: Union[str, None] = None
+    disabled: Union[bool, None] = None
```

Comment:
Can we have a DocString explaining the Users class please. 
Explaining that it is used for authentication rather than as a regular part of the API.

E.g.
```
class Users(SQLModel, table=True):
    """Users are required to be authenticated to use some functionality of the API."""
    username: str = Field(primary_key=True)
    hashed_password: Union[str, None] = None
    email: Union[str, None] = None
    full_name: Union[str, None] = None
    disabled: Union[bool, None] = None   # If the user is disabled they will be unable to authenticate
```